cogs/logging_cog.py

```diff
@@ -106,6 +126,23 @@ async def on_invite_delete(self, invite):
         # TODO Implement
         pass
 
+    @tasks.loop(seconds=VOICE_CHECK_FREQUENCY)
+    async def voice_channel_check(self):
+        """
+        Task loop that updates the time users have spent
+        in voice chat.
+        """
+
+        for guild in self.bot.guilds:
+            for channel in guild.voice_channels:
+                for member in channel.members:
+                    Voice.first_or_create(user_id=member.id,
```

Comment:
Why didn't you just use the object returned from this method call for the incrementation?

Reply:
Tried that, wasn't working for me

Reply:
Talked with Will and tried a different approach but it didn't work, can revisit if needed.